### DISCUSSION TITLE
israelhayom.co.il: ads

### DISCUSSION THREAD

**Discussion by Hebrew-uBO-User** [uBlockOrigin/uAssets#24686] [Q&A] [answered]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
https://www.israelhayom.co.il/
```

### Description

Firefox for **Android** + uBlock Origin + EasyList **Hebrew**

1. Open **any article**.
2. There is a widget at the bottom.
3. There is a blank space on the widget.

I want to hide the blank space. I don't want to hide the whole widget.
The built-in adblocker in Opera for Android hides it, so I guess it's possible.
What do you suggest?

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>
<summary>Details</summary>

```yaml


```
</details>

**Comment by stephenhawk8054** [comment]
Can you screenshot and describe based on that screenshot?

**Comment by ghajini** [comment, marked as answer]
use `:style` if you are talking about this space

https://github.com/user-attachments/assets/9a8a9f01-30fc-43ca-b909-fb10977e869b)

```
israelhayom.co.il##.new-toolbar:style(padding-bottom: 0px !important;)
israelhayom.co.il##.new-toolbar:style(height: 50px !important;)
israelhayom.co.il##.footer:style(padding-bottom: 50px !important;)
```

**Comment by Hebrew-uBO-User** [comment]
Fixed: https://github.com/easylist/EasyListHebrew/commit/509b8a6b97c9aa7ea36d63507648fc856c227e8d
Thank you!
